audit-tools/ons-audit-uri/main.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-// NOTE: Any changes to the ONS website might stop this App working and it will need adjusting ...
+// NOTE: Any changes to the ONS website might stop this App working and it will need adjusting ..
```

Comment:
I'm not sure if it was the linting command, but it seems all of the elipsis have lost their last dot 😆

Reply:
answered in slack